src/3d/castlecameras.pas

```diff
@@ -1249,6 +1250,10 @@       TExamineVectors = record
     { How fast user moves the scene by pressing keys. }
     property KeysMoveSpeed: Single read FKeysMoveSpeed write FKeysMoveSpeed default 1.0;
 
+    { Optional movement algorithm, moves the camera exactly as many units as
+      the mouse position has changed especially useful in 2D in the editor. }
+    property ExactMovement: Boolean read FExactMovement write FExactMovement default false;
```

Comment:
I think you want to remove this declaration. `ExactMovement` is now declared inside `TCastle2DNavigation`.

I guess these changes remained because you initially made it as a modification of `TCastleExamineNavigation` and only later reworked to be a new class :)

Reply:
That was invalid comment from me, closing :) All is OK, TCastle2DNavigation just changes default.